CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.21.1-0.7.44a] - 2025-01-14
```

Comment:
actually, I will invoke patchLog in the action, so this will be duplicate?

Reply:
I will bump the version and write a changelog, so that's okay